### PR TITLE
Align error logs text color with theme

### LINKED ISSUE #1631: Theme misalignment in error logs summary
### Dev Home version

0.503.261.0

### Windows build number

10.0.22621.2283

### Other software

_No response_

### Steps to reproduce the bug

1. Go to Machine Configuration > Clone Repositories.
2. Click on "Add Repository."
3. Provide a repository and specify a location where a folder with the same name already exists (to deliberately trigger a cloning failure).
4. Setup the repository.
5. After the cloning process fails, navigate to the Summary page.
6. Review the error logs heading and caption. If you're using the light mode, they will align with the overall theme. However, in dark mode, the text appears dark and difficult to read.

### Expected result

The text color should align with the selected theme.

### Actual result

The text color aligns with the Light theme but not with the dark theme

image

image

### Included System Information

Windows: Windows.Desktop v10.0.22621.2283
System Architecture: X64

### Included Extensions Information

_No response_

### PR DESCRIPTION
**Pull request by hamnarauf**
## Summary of the pull request
The text color for the heading and caption of error logs in the summary section of cloned repositories has been adjusted to match the selected theme.

## References and relevant issues
https://github.com/microsoft/devhome/issues/1631

## Validation steps performed
Setup a repository with failed cloning and observed the summary page in light and dark mode.

Dark theme:
image

Light theme:
image

## PR checklist
- [x] Closes #1631
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml [hunk 1]
```diff
@@ -210,10 +210,10 @@
                                     <RowDefinition/>
                                     <RowDefinition/>
                                 </Grid.RowDefinitions>
-                                <TextBlock Text="Error logs" Grid.Row="0" Grid.Column="0" Margin="0, 5, 0, 0" Foreground="{StaticResource TextFillColorPrimary}"/>
+                                <TextBlock Text="Error logs" Grid.Row="0" Grid.Column="0" Margin="0, 5, 0, 0"/>
                                 <StackPanel Orientation="Horizontal" Spacing="5" Grid.Row="1" Grid.Column="0">
-                                    <TextBlock Text="{x:Bind ViewModel.FailedTasks.Count}" Foreground="{StaticResource TextFillColorSecondary}"/>
-                                    <TextBlock Text="installation errors" Foreground="{StaticResource TextFillColorSecondary}"/>
+                                    <TextBlock Text="{x:Bind ViewModel.FailedTasks.Count}" Foreground="{ThemeResource TextFillColorSecondary}"/>
+                                    <TextBlock Text="installation errors" Foreground="{ThemeResource TextFillColorSecondary}"/>
```
Comment (bbonaby) [inline tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml:216]: Thanks for this @hamnarauf.

Looks good, but since you're doing this already, I see that on line 230 in this file we're also using StaticResource instead of the ThemeResource for the foreground of the 'MessageToShow' textblock.  Would you be able to update that as well? 

Thanks again!

Comment (hamnarauf) [reply tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml:216]: Made `MessageToShow` use `ThemeResource`

**Review by bbonaby — APPROVED** [review]
Thanks @hamnarauf

File: tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml [hunk 2]
```diff
@@ -210,10 +210,10 @@
                                     <RowDefinition/>
                                     <RowDefinition/>
                                 </Grid.RowDefinitions>
-                                <TextBlock Text="Error logs" Grid.Row="0" Grid.Column="0" Margin="0, 5, 0, 0" Foreground="{StaticResource TextFillColorPrimary}"/>
+                                <TextBlock Text="Error logs" Grid.Row="0" Grid.Column="0" Margin="0, 5, 0, 0"/>
```
Comment (krschau) [inline tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml:213]: [Not part of this PR, but noticing it now] @dhoehna or @AmelBawa-msft, whoever owns this page -- is the text "Error logs" here and "View log" below a problem? Should this be localized?

**Review by krschau — APPROVED** [review]

File: tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml (hunk 2 above)
Comment (AmelBawa-msft) [reply tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml:213]: I would say it should be localizable, unless that's a debug feature only? Adding @dhoehna for context.

Comment (dhoehna) [reply tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml:213]: Yes, it should be localizable.  That was my mistake.

Comment (krschau) [reply tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml:213]: Opened #1670
